app/main.py

```diff
@@ -2,0 +1,12 @@
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command, source, destination = command.split()
+    if command != "mv":
+        return
+    path_, new_file_name = os.path.split(destination)
+    if path_ and not os.path.exists(path_):
+        os.makedirs(path_)
```

Comment:
Code Efficiency: If the directory already exists, `os.makedirs` will raise an error. You should use `os.makedirs(path_, exist_ok=True)` to prevent this error.